rakefile

```diff
@@ -9,6 +11,56 @@ include PerfResultsHelper
 
 REF_ARCH_STD, REF_ARCH_LARGE = ["S","L"]
 
+# Create a trumped up class to get a Beaker helpers outside of a Beaker run
+class ClassMixedWithDSLHelpers
+  include Beaker::DSL::Helpers
+  include Beaker::DSL::Wrappers
+  include Beaker::DSL::Roles
+  include Beaker::DSL::Patterns
+
+  def options
+    { cache_files_locally: true }
+  end
+
+  def logger
+    Beaker::Logger.new
+  end
+end
+
+def master_from_beaker_hosts_hash(hosts)
+  master = nil
+  hosts.each_key do |name|
+    host = hosts[name]
+    master = host if host[:roles].include?("master")
+  end
+  master
+end
+
+def fetch_pe_for_master(beaker_proxy, master, ver, dir, dest = "/tmp")
+  local = File.directory?(dir)
+  filename = "puppet-enterprise-#{ver}-#{master['packaging_platform']}"
+  if local
+    extension = File.exist?("#{dir}/#{filename}.tar.gz") ? ".tar.gz" : ".tar"
+    non_exist_msg = "PE #{dir}/#{filename}#{extension} does not exist"
```

Comment:
should probably say .tar or .tar.gz in the error message, since neither exist, and either is allowed... otherwise someone reading it might think they need to gzip it and put it there... when really the code will gzip it if needed.